Description: Fix man pages typos
 Fix typos in the some man pages
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Last-Update: 2019-10-02

--- a/doc/man/man5/slurm.conf.5
+++ b/doc/man/man5/slurm.conf.5
@@ -3578,7 +3578,7 @@
 Specifies the nodes which are to not be placed in power save mode, even
 if the node remains idle for an extended period of time.
 Use Slurm's hostlist expression to identify nodes with an optional ":" separator
-and count of nodes to exclude from the preceeding range.
+and count of nodes to exclude from the preceding range.
 For example "nid[10-20]:4" will prevent 4 usable nodes (i.e IDLE and not 
 DOWN, DRAINING or already powered down) in the set "nid[10-20]" from being
 powered down.
--- a/doc/man/man1/sbatch.1
+++ b/doc/man/man1/sbatch.1
@@ -159,7 +159,7 @@ vertical bar separators. For example:
 The \-\-batch argument must be a subset of the job's
 \fB\-\-constraint\fR=<\fIlist\fR> argument (i.e. the job can not request only
 KNL nodes, but require the script to execute on a Haswell node).
-If the request can not be satisifed from the resources allocated to the job,
+If the request can not be satisfied from the resources allocated to the job,
 the batch script will execute on the first node of the job allocation.
 
 .TP
--- a/doc/man/man1/scontrol.1
+++ b/doc/man/man1/scontrol.1
@@ -236,7 +236,7 @@ rebooted and returned to service "As Soo
 defined message can be added to the nodes by using the "reason" option. The
 message will be cleared when the node is resumed.  If "nextstate" is specified
 as "DOWN", then the node(s) will remain in a down state after rebooting. If the
-"nextstate" is specifed as "RESUME", then the node(s) will resume as normal
+"nextstate" is specified as "RESUME", then the node(s) will resume as normal
 when the node registers. "Resuming" nodes will be considered as available in
 backfill future scheduling and won't be replaced by idle nodes in a reservation.
 When using the "nextstate" and "reason" options together the reason will be
@@ -626,7 +626,7 @@ Only pending jobs can have the deadline
 Only the Slurm administrator or root can change this parameter.
 .TP
 \fBDelayBoot\fP=<time_spec>
-Change the time to decide wether to reboot nodes in order to satisfy job's
+Change the time to decide whether to reboot nodes in order to satisfy job's
 feature specification if the job has been eligible to run for less than this
 time period. See salloc/sbatch man pages option \-\-delay\-boot.
 .TP
@@ -807,7 +807,7 @@ Set the job's QOS (Quality Of Service) t
 Value may be cleared with blank data value, "QOS=".
 .TP
 \fBReboot\fP=<yes|no>
-Set the job's flag that specifies wether to force the allocated nodes to reboot
+Set the job's flag that specifies whether to force the allocated nodes to reboot
 before starting the job. This is only supported with some system configurations
 and therefore it could be silently ignored.
 .TP
--- a/doc/man/man5/acct_gather.conf.5
+++ b/doc/man/man5/acct_gather.conf.5
@@ -191,7 +191,7 @@ ProfileInfluxDBDatabase option.
 
 .TP
 \fBProfileInfluxDBUser\fR
-Optinal InfluxDB username that should be used to gain access to the database
+Optional InfluxDB username that should be used to gain access to the database
 configured in ProfileInfluxDBDatabase. This is only needed InfluxDB is
 configured with authentication enabled in the [http] config section and a user
 has been granted at least WRITE access to the database. See also
@@ -229,7 +229,7 @@ respectively or dynamically through scon
 .TP
 NOTE:
 Perhaps it's a good idea to use a monitoring and analytics tool such as Grafana
-on top of InfluxDB. This kind of tools permit to create dashboards, tables, and
+on top of InfluxDB. This kind of tools permit one to create dashboards, tables, and
 other graphics using the stored time series. This way, it is easier to correlate
 resource usage peaks reported by other node monitoring tools such as Ganglia
 with specific job step tasks.
